code/modules/admin/verbs/borgpanel.dm

```diff
@@ -1,15 +1,23 @@
-/datum/admins/proc/open_borgopanel(borgo in GLOB.silicon_mobs)
+/datum/admins/proc/open_borgopanel(mob/living/silicon/robot/borgo as null|anything in GLOB.cyborg_list)
 	set category = "Admin"
 	set name = "Show Borg Panel"
 	set desc = "Show borg panel"
 
 	if(!check_rights(R_ADMIN))
 		return
 
-	if (!istype(borgo, /mob/living/silicon/robot))
-		borgo = input("Select a borg", "Select a borg", null, null) as null|anything in sort_names(GLOB.silicon_mobs)
-	if (!istype(borgo, /mob/living/silicon/robot))
-		to_chat(usr, "<span class='warning'>Borg is required for borgpanel</span>")
+	if(!length(GLOB.cyborg_list))
+		to_chat(usr, "<span class='warning'>There are no borgs to show a panel for!</span>")
+		return
+
+	if(!istype(borgo))
+		borgo = input("Select a borg", "Select a borg", null, null) as null|anything in sort_names(GLOB.cyborg_list)
+		if(!borgo)
+			return
+
+	if(QDELING(borgo))
```

Comment:
somewhere here needs `if(istype(borgo))` here

Reply:
That's what the !istype check was supposed to be for